tests/phenotype/test_phenotype_resources.py

```diff
@@ -0,0 +1,395 @@
+            'phenotype': 'Hand Tremor',
+            'hpo_id': 'HP:0002378',
+            'observed': 'positive',
+            'participant_id': ph1['participant_id']
+        }
+        # Send post request
+        response = self.client.post(url_for(PHENOTYPES_LIST_URL),
+                                    headers=self._api_headers(),
+                                    data=json.dumps(ph2))
+        # Check status code
+        self.assertEqual(response.status_code, 201)
+        # Check database
+        phe = Phenotype.query.count()
+        self.assertEqual(phe, 2)
+        pph = Participant.query.all()[0].phenotypes
+        self.assertEqual(len(pph), 2)
+
+    def test_get(self):
+        # Create and save phenotype to db
+        kwargs = self._create_save_to_db()
+        # Send get request
+        response = self.client.get(url_for(PHENOTYPES_URL,
+                                           kf_id=kwargs['kf_id']),
+                                   headers=self._api_headers())
+
+        # Check response status code
+        self.assertEqual(response.status_code, 200)
+        # Check response content
+        response = json.loads(response.data.decode('utf-8'))
+        phenotype = response['results']
+        participant_link = response['_links']['participant']
+        participant_id = urlparse(participant_link).path.split('/')[-1]
+        self.assertEqual(phenotype['kf_id'], kwargs['kf_id'])
+        self.assertEqual(participant_id,
+                         kwargs['participant_id'])
+        self.assertEqual(phenotype['phenotype'], kwargs['phenotype'])
+        self.assertEqual(kwargs['hpo_id'],
+                         phenotype['hpo_id'])
+        self.assertEqual(kwargs['observed'], phenotype['observed'])
+        self.assertEqual(phenotype['age_at_event_days'],
+                         kwargs['age_at_event_days'])
+        self.assertEqual(participant_id, kwargs['participant_id'])
+
+    def test_get_not_found(self):
+        """
+        Test get phenotype that does not exist
+        """
+        # Create phenotype
+        kf_id = 'non_existent'
+        response = self.client.get(url_for(PHENOTYPES_URL, kf_id=kf_id),
+                                   headers=self._api_headers())
+        self.assertEqual(response.status_code, 404)
+        response = json.loads(response.data.decode("utf-8"))
+        message = "could not find phenotype `{}`".format(kf_id)
+        self.assertIn(message, response['_status']['message'])
+
+    def test_get_all(self):
+        """
+        Test retrieving all phenotypes
+        """
+        kwargs = self._create_save_to_db()
+
+        response = self.client.get(url_for(PHENOTYPES_LIST_URL),
+                                   headers=self._api_headers())
+        self.assertEqual(response.status_code, 200)
+        response = json.loads(response.data.decode("utf-8"))
+        content = response.get('results')
+        self.assertEqual(len(content), 1)
+
+    def test_patch(self):
```

Comment:
After you update the patch() method in the phenotype resource, this test will need to be updated based on expected PATCH behavior. It should test that only the properties supplied in body were updated on the phenotype and the rest of the properties remained unchanged since PATCH allows for partial update